src/databricks/labs/ucx/hive_metastore/hms_lineage.py

```diff
@@ -0,0 +1,43 @@
+import base64
+import time
+
+from databricks.sdk import WorkspaceClient
+from databricks.sdk.service.compute import GlobalInitScriptDetailsWithContent
+
+
+class HiveMetastoreLineageEnabler:
+    def __init__(self, ws: WorkspaceClient):
+        self._ws = ws
+
+    def check_lineage_spark_config_exists(self) -> GlobalInitScriptDetailsWithContent:
+        for script in self._ws.global_init_scripts.list():
+            gscript = self._ws.global_init_scripts.get(script_id=script.script_id)
+            if gscript:
```

Comment:
Nit: Too much level of nesting, rewrite with "if gscript is None: continue"

Reply:
Took care of the same.